providers/certification-client/units/client-cert-odm-desktop-22-04.pxu

```diff
@@ -209,6 +210,7 @@ exclude:
 
 id: client-cert-odm-desktop-22-04
 _name: DO NOT USE - desktop-22-04 - placeholder to establish that total number of tests has not changed
+    after-suspend-wwan-automated
```

Comment:
`after-suspend-wwan-automated` should not be added here

Reply:
Fixed by removing the other call to `after-suspend-wwan-automated` that was called too early.